src/patient-registration/patient-registration.component.tsx

```diff
@@ -532,6 +533,7 @@ export const PatientRegistration: React.FC = () => {
               <Row>
                 <Column lg={2} md={2} sm={1}>
                   <div className={styles.fixedPosition}>
+                    hello world
```

Comment:
let me remove this, just wanted to see if local development was working (and it was!)